03_Collection Types.playground/section-1.swift

```diff
@@ -46,6 +47,10 @@ var array = [Int](count: 3, repeatedValue: 0) //Initalizes an array of lenght 3
 
 var compundArray = array + emptyArray
 
+var reverseShoppingList: [String] = shoppingList.reverse()
+
+reverseShoppingList.removeLast() // Remove last item, the other way with removeFirst()
+reverseShoppingList.popLast() // Optional remove last item so that won't crash when array is empty, don't have popFirst()
```

Comment:
Yes, example with `var array: [String?] = []`, if use `removeLast()`, the app will crash, but doesn't with `popLast()` (return nil) because it create optional array in return, not implicity unwraped